src/gettingstarted/cli.md

```diff
@@ -97,3 +97,30 @@ Once Bash for Windows is installed, you can install the Platform.sh CLI with the
 ```bash
 curl -sS https://platform.sh/cli/installer | php
 ```
+
+## API Tokens
+
+The CLI can also be used from 3rd party CI services or other automation tools, and supports an API Token authentication option for that purpose.
+
+An API token can be created through the UI.  Go to the "User" page from your accounts drop-down, then select the "Account Settings" tab, then the "API Tokens".
+
+Click the "Create an API Token" link.  You may be asked to reverify your password.
+
+Enter a unique application name to identify the token.  The API token can authorize the CLI to access project information, or project information and account information.  In most cases it's not necessary to give the CLI access to change account settings so that checkbox can be unchecked.  
```

Comment:
Hm. I think this scope option is actually totally irrelevant, for now.

Reply:
Well, the checkbox is there so we should say something about it.  What should we say if not the above?